src/App.js

```diff
@@ -1,5 +1,25 @@
+import { parsePlayers, parseRound } from './parseInput';
+import { processInputToOutput } from './Simulate';
+import { formatOutput } from './formatOutput';
+
+import { MissionUtils } from '@woowacourse/mission-utils';
+const Console = MissionUtils.Console;
+
+const Message = {
+  playerList:
+    '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n',
+  roundNumber: '시도할 횟수는 몇 회인가요?\n',
+};
```

Comment:
```suggestion
const MESSAGES = {
  PLAYER_LIST:
    '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n',
  ROUND_NUMBER: '시도할 횟수는 몇 회인가요?\n',
};
```
지금은 파스칼 케이스로 작성해주셨는데 이 메세지는 런타임 중에는 변하지 않는 것 같아서 상수 네이밍인 스네이크 케이스로 작성하는건 어떨까요??

Reply:
아차, 스네이크 케이스는 코딩 스타일 가이드에도 있었죠ㅠㅠ 여기를 고치는 걸 생각을 못했습니다